api-generation/build.gradle

```diff
@@ -8,6 +8,9 @@ dependencies {
   implementation("ai.h2o:h2o-algos:${h2oVersion}")
   implementation("ai.h2o:h2o-automl:${h2oVersion}")
   implementation("ai.h2o:h2o-ext-xgboost:${h2oVersion}")
+  compileOnly("org.apache.spark:spark-core_${scalaBaseVersion}:${sparkVersion}")
```

Comment:
Actually only one is required, thanks for checking that. I need `org.apache.spark:spark-mllib` because I use the `Params` class at https://github.com/h2oai/sparkling-water/blob/5cafdc31603d6cb4c4e15687fdc3066566864e62/api-generation/src/main/scala/ai/h2o/sparkling/api/generation/python/Word2VecTemplate.scala#L20

Reply:
ok, thanks!